index.html

```diff
@@ -171,6 +174,35 @@ <h1>Profesional Art Printing Data</h1>
                 <button class="see-project-button" type="button">See Project</button>
             </div>
         </div>
+        <div class="project-details-modal-background disappear">
+            <div class="project-details-modal">
+                <div class="project-modal-header">
+                    <div class="modal-head">
+                        <h3 class="modal-heading-mobile">Multi Post Stories</h3>
+                        <h3 class="modal-heading-desktop">Keeping track of hundreds  of components website</h3>
+                        <i class="dark-cross-button"></i>
+                    </div>
+                    <ul class="languages">
+                        <li><span>html</span></li>
+                        <li><span>Bootstrap</span></li>
+                        <li><span>Ruby on rails</span></li>
+                    </ul>
+                </div>
+                <div class="modal-content">
+                    <img src="images/portfolio/snapshoot-portfolio.png" alt="Portfolio snapshot image">
+                    <div class="modal-lower-half">
+                        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                            been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            galley of type and scrambled it to make a type specimen book. It has survived not only five
+                            centuries, but also the leap into electronic typesetting, remaining essent</p>
+                        <div class="modal-buttons">
+                            <div class="btns">See Live</div>
+                            <div class="btns">See Source</div>

```

Comment:
- [ ] Kindly make sure to add the icons on those button elements to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/161792019-515e00dc-ed53-4cdc-bd97-d86b6556f21e.png) | ![image](https://user-images.githubusercontent.com/62290545/161791974-ae275025-7329-4a10-9ac0-2cac252dc6af.png) |

Reply:
All good now 👍🏽